src/index.html

```diff
@@ -25,20 +25,20 @@ <h2 class="h2-heading">a JW Marriott Product</h2>
         <div class="past-future-heading">
           <h2>past & future reservations</h2>
         </div>
-        <div class="past-future-card-area">
-        </div>
+        <div class="past-future-card-area"></div>
       </div>
 
       <div class="book-a-cabin-container hidden">
         <div class="past-future-heading">
           <h2>book a new reservation</h2>
         </div>
         <div class="select-dates-container">
-          <select class="possible-dates-dropdown dates">
+          <input type="text" class="date-input" placeholder="year/month/day">
+          <!-- <select class="possible-dates-dropdown dates">
```

Comment:
Consider removing commented out code if you have already solved the problem you were using it for in the first place.

Reply:
Thanks for finding this. Yes, that comment is no longer relevant. I will remove it.